Fika.Core/Utils/WorldToScreen.cs

```diff
@@ -0,0 +1,77 @@
+﻿using EFT.Animations;
+using EFT.CameraControl;
+using EFT.InventoryLogic;
+using Fika.Core.Coop.Players;
+using UnityEngine;
+
+namespace Fika.Core.Utils
+{
+    public static class WorldToScreen
+    {
+        public static bool GetScreenPoint(Vector3 worldPosition, CoopPlayer mainPlayer, out Vector3 screenPoint)
+        {
+            CameraClass worldCameraInstance = CameraClass.Instance;
+            Camera worldCamera = worldCameraInstance.Camera;
+
+            screenPoint = Vector3.zero;
+
+            if (mainPlayer == null || worldCamera == null)
```

Comment:
Could we change to, and in other places as well
```cs
if (mainPlayer == null || worldCamera == null)
{
    return false;
}
```
Just to keep scopes consistent throughout the repo, I've been cleaning up old code recently to achieve the same.